server.py

```diff
@@ -313,11 +315,10 @@ def handle_tile(z, x, y, fmt, tile_pixel_size=None):
     tile_size = tile_pixel_size / 256
     if tile_size != int(tile_size):
         return abort(400, "Invalid tile size. %s is not a multiple of 256." % tile_pixel_size)
-
-    requested_tile = TileRequest(z, x, y, fmt)
-
     tile_size = int(tile_size)
 
+    requested_tile = TileRequest(z, x, y, tile_size, fmt)
```

Comment:
If `tile_size` is now part of `TileRequest`, do we need to pass it in separately to `meta_and_offset`? I guess what I'm asking is whether there's a difference between `TileRequest.scale` and `tile_size`, or are the two always going to be the same?

Reply:
Nope, don't need to pass it in. I did this for the tapalcatl2 path but forgot to clean it up in the metatile path.